pages/about2.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="../styles/style-flexbox.css">
+  <title>About</title>
+</head>
+
+<body>
+
+  <header class="portfolio">
+    <h1 class="title">
+      Camilla Berretta, full-stack developer
+    </h1>
+  </header>
+
+  <main>
+    <section>
+      <h1 class="othertitle">
+        ABOUT
+      </h1>
```

Comment:
It depends on who you ask, but your document should have only one h1 element. Of course you can add multiple and the browser will still render the h1 tags, but it is more semantically correct to only use 1 since heading tags have a hierarchy

Since you already have an h1 element on lines 15-17, you can use a h2 here.

Also, it's perfectly fine to have opening tag, content, and closing tag all on one line if it's not too long, like:
```HTML
<h1 class="othertitle">ABOUT</h1>